### PR TITLE
feat(op sets): update stakes when forceUnregister

### PR DESCRIPTION
**Pull request by stevennevins**

### CODE REVIEW

File: src/StakeRegistry.sol [hunk 1]
```diff
@@ -148,6 +150,12 @@ contract StakeRegistry is StakeRegistryStorage {
     ) external onlyRegistryCoordinator returns (uint192) {
         uint192 quorumsToRemove;
 
+         address avs = address(
```
Comment (ypatil12) [inline src/StakeRegistry.sol]: Could make this immutable & refer to it as serviceManager?

Comment (ypatil12) [reply src/StakeRegistry.sol]: similar to what we do in registry coordinator

**Review by ypatil12 — COMMENTED** [review]
One comment

File: src/StakeRegistry.sol (hunk 1 above)
Comment (stevennevins) [reply src/StakeRegistry.sol]: yeah good call

Comment (stevennevins) [reply src/StakeRegistry.sol]: all set [9b96076](https://github.com/Layr-Labs/eigenlayer-middleware/pull/300/commits/9b96076c9460c37ba7bdab4a7be32042890380e8)

**Review by ypatil12 — APPROVED** [review]

**Pull request merged**